frontend/app/components/ui/Icons/Or_spot_outline.tsx

```diff
@@ -0,0 +1,19 @@
+
+/* Auto-generated, do not edit */
+import React from 'react';
```

Comment:
you generated the icon but looks like you've forgot to add the .svg file to git and commit it so it will be removed as soon as someone will pull the code and run gen:icons